packages/data-point/lib/entity-types/base-entity/resolve.test.js

```diff
@@ -146,17 +146,17 @@ describe('ResolveEntity.resolveEntity', () => {
   }
 
   test('It should resolve entity', () => {
-    return resolveEntity('hash:asIs', 'foo').then(acc => {
+    return resolveEntity('model:asIs', 'foo').then(acc => {
```

Comment:
using `model:asIs` because these `hash:asIs` functions are not receiving objects